Gems/AWSCore/Code/Source/Framework/Platform/Android/ResolveCaCertBundlePath_Android.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/PlatformDef.h>
+// The AWS Native SDK AWSAllocator triggers a warning due to accessing members of std::allocator directly.
+// AWSAllocator.h(70): warning C4996: 'std::allocator<T>::pointer': warning STL4010: Various members of std::allocator are deprecated in
+// C++17. Use std::allocator_traits instead of accessing these members directly. You can define
+// _SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING or _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS to acknowledge that you have received
+// this warning.
+AZ_PUSH_DISABLE_WARNING(4251 4996, "-Wunknown-warning-option")
+#include <aws/core/utils/memory/stl/AWSString.h>
+AZ_POP_DISABLE_WARNING
+#include <AzCore/Android/Utils.h>
+#include <AzCore/std/string/string.h>
+
+namespace AWSCore
+{
+    namespace Platform
+    {
+        void ResolveCaCertBundlePath(Aws::String& caCerBundlePath)

```

Comment:
Minor: it is more intuitive to return the path instead of passing path as parameter reference, so the function can be GetCACertBundlePath

Reply:
Will address in future PR when we start the testing for the GameLift gem.

Reply:
Addressed in the latest commit.